chapter10/sxg/ex_10_2.pl

```diff
@@ -0,0 +1,26 @@
+#!perl
+use warnings;
+use strict;
+use 5.010;
+
+my $secret = int(1 + rand 100);
+my $count =0;
+
+my $debug = $ENV{DEBUG} // 1;
+print "The secret number is :$secret\n" if $debug;

```

Comment:
I agree.. I was just curious about perl. This cannot be done in any of the C like language. :)